ledger/acctupdates.go

```diff
@@ -1367,6 +1367,40 @@ func (au *accountUpdates) lookupResource(rnd basics.Round, addr basics.Address,
 	}
 }
 
+func (au *accountUpdates) lookupAccountDeltas(rnd basics.Round) (ledgercore.AccountDeltas, error) {
+	au.accountsMu.RLock()
+	defer au.accountsMu.RUnlock()
+	var offset uint64
+	var deltas ledgercore.AccountDeltas
+	offset, err := au.roundOffset(rnd)
+	if err != nil {
+		return deltas, err
+	}
+	if uint64(len(au.deltas)) < offset || offset == 0 {
```

Comment:
`uint64(len(au.deltas)) < offset` this check is already done in `roundOffset` and we are under a lock